tests/library/debug-controller.spec.ts

```diff
@@ -48,10 +48,11 @@ const test = baseTest.extend<Fixtures>({
   },
   connectedBrowserFactory: async ({ wsEndpoint, browserType }, use) => {
     const browsers: BrowserWithReuse [] = [];
-    await use(async () => {
+    await use(async launchOptions => {
```

Comment:
- I don't see `launchOptions` being used anywhere.
- You probably want `{ ..._defaultLaunchOptions, ...launchOptions }` instead.